### ISSUE TITLE
Abandoned drive through map gen

### ISSUE THREAD

**Issue by Spacer-1970** [Noctifer-de-Mortem/nocts_cata_mod#493]
Sometimes the overworld map generates abandoned drive through tiles and one time when I was the crop fields I quickloaded my save and then on the overmap the dirt road tiles in the fields turned to abandoned drive through tiles

**Comment by chaosvolt** [comment]
Are you sure this is at all related to Cata++? That sounds like some kinda save/load bug on DDA's end. I assume you mean the abandoned farm overmap special, which is vanilla.

**Comment by Spacer-1970** [comment]
It could be possible that dda just had an error in save loading but is it
possible to revert such an error? And sorry for bothering you if this error
didn't concern your mod.

**Comment by chaosvolt** [comment]
Not sure how to fix that to be honest, definitely check if any similar issues have been opened in DDA perhaps.

**Comment by Spacer-1970** [comment]
Yeah I will probably check there in due time but thanks for your help!
